### DISCUSSION TITLE
Lets think big - We need to stablish Iranian underground state

### DISCUSSION THREAD

**Discussion by TheMacroLady** [WeAreMahsaAmini/Core#107] [Ideas]
We are living in the 21 century and we should come up with new strategies of civil resistance. Let's show the world who we are. We need to establish a legitimate virtual state to build our future country and even vote for it.
- designing an app to run surveys and see what our nation wants. e.g. referendum
- humans are herd animals and they need a leader. How we can find and link our big political brains ?
- better coordination. how to link people when the internet is down
- supporting financially the families who are affected
- asking world leaders to support our movement financially, socially, and politically: eg. air lifting starlink antenna. a
- launching a cyber war against Basij institutions - can we disturb their communication during mass protests?
- using other tactics like organizing hunger strikes, and road blockades.
- making a database that is helping Iranian ( not only influencers but also business leaders, political leaders, news agencies, number of #s etc.)
- designing aerial vehicles to help protesters on movements of special forces
- live protest tracker - estimation of size ( protester and special forces )
- connecting with social activists from across the world to raise our voices
- how people inside the Basij institution can leak information without risking their identity?
- making a database getting the name of the people in special forces( by friends and family, hacking the system - internal leaks )
- making a database and identifying the children of Iranian fucking leaders who live overseas and estimating their wealth based on their lifestyle

**Comment by AnonymousMaximus2** [comment]
The idea is great but

but

you need infrastructure for this idea,

If you create illegal or unrecognized underground group, that does not supported by a government and not organized well, your team an not do great steps.

and I explained this infrastructure here:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Reply by sinahnyazdi** [reply]
This Idea is very practical and I like it too.
I suggested the money for it here #114

**Comment by kaveh6** [comment]
I really like the idea of having a block-chain based voting system for various reasons . we could come up with an application so that every one can see different surveys and go vote in a non-corruptive system .

**Comment by AnonymousMaximus2** [comment]
Block-chain , NFT , Metaverse are good ideas, but they need money and workforce and strong management, and most important is they need more time maybe years to develop this idea, and at this time we do not have enough time.

but we can start here: 

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by babooloo** [comment]
Great collection of big ideas. I like the one about running surveys. There needs to be a system to help find out the preferences/wants/needs of the majority of the population.

**Reply by babooloo** [reply]
Here is an idea I came across regarding surveys. It's not s survey exactly, but it is a way to amplify good ideas regarding governance of a country that we can get inspiration from.
https://bugs.telegram.org/c/20948
